aries-site/src/examples/components/avatar/AvatarDarkModeExample.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Avatar, Box } from 'grommet';
+
+export const AvatarDarkModeExample = () => (
+  <Box
+    width="medium"
+    height='small'
+    background={{ color: 'background', dark: true }}
+    justify='center'
+    align='center'
+  >
+    <Avatar
+      background={{ color: 'background', dark: false }}
+      alt="Ctera Logo"
+      src="/ctera.svg"
+      size="large"
+      round="medium"
+      align="undefined"
```

Comment:
```suggestion
```
I've never seen this before, can we remove?

Reply:
I need it because grommet Avatar sets this by default `align:"center"`